src/pages/MFARestoreScreen.vue

```diff
@@ -146,6 +150,7 @@ async function handleLocalRecovery(key: string) {
   storage.session.setIsLoggedIn()
   user.setUserInfo(userInfo)
   user.setLoginStatus(true)
+  console.log(app.curve, app.chainType)
```

Comment:
Check if these logs are necessary, use `devLogger.log` if you want these logs only in dev

I've created a util `@/utils/devLogger`